scripts/apidoc/typedoc.ts

```diff
@@ -286,6 +286,18 @@ export function extractDeprecated(
   return deprecated.length === 0 ? undefined : deprecated;
 }
 
+/**
+ * Extracts the "throws" tag from the provided signature.
+ *
+ * @param reflection The reflection to check.
+ *
+ * @returns The message explaining the error in the @throws tag, otherwise `undefined`.
```

Comment:
```suggestion
 * @returns The message explaining the conditions when this method throws. Or `undefined` if it does not throw.
```

Not sure about the wording.